README.md

```diff
@@ -3,6 +3,9 @@ A place to organize style guides, best practices, tools, and techniques for Stan
 
  - [Best Practices](/best-practices)
    - [Documentation](/best-practices/documentation)
+   - [How To - Readme](/best-practices/howto_readme.md)
+   - [Licensing](/best-practices/licensing.md)
+   - [Version Control](/best-practices/version_control.md)
```

Comment:
I think this has merit, but I was torn between the maintainability issue, and newcomers coming to the document and being able to know what it all contained. Maybe this will become clear as things develop further ? :confused: